### ISSUE TITLE
the throttling time doubled when one URL pattern try to throttling 2 requests

### ISSUE THREAD

**Issue by joychester** [severest/url-throttler#1]
for example the page have 2 different URLs : 
**Request 1**: https://host.com/path/to/url/v3?query=abc
**Request 2**: https://host.com/path/to/url/v3?query=bcd

using URL-throttler with match pattern in following:
**https://host.com/path/to/url/v3***

The expected result: 
Both Request 1 and Request 2 will be blocked by 5 seconds

The actual result: 
The Request 1 will be blocked by 5 seconds, the Request 2 will be blocked by 10 seconds.

**Comment by severest** [comment]
Yes, I'm able to reproduce this. It seems to be related to the way [chrome.webRequest](https://developer.chrome.com/extensions/webRequest) listeners work. Uncertain how to improve this issue at the moment.

A work-around would be to add two separate, more unique URLs (one item for each request) rather than using the wildcard.

**Comment by severest** [comment]
Actually nvm, that doesn't work either...
